intlist.c

```diff
@@ -0,0 +1,269 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// intlist.c
+// Program on main branch.
+// ------------------------------------------------------------------------------------------------
+
+
+#include <stdio.h>
+#include <malloc.h>
+#include "intlist.h"
+
+
+struct node {

```

Comment:
Whenever you define a new type (struct in this case) or function, you should name it with a capital letter